recipe/meta.yaml

```diff
@@ -37,9 +37,7 @@ outputs:
     script: clean_python.sh
     test:
       commands:
-        - test -f ${PREFIX}/lib/libChronoEngine.so  # [linux]
-        - test -f ${PREFIX}/lib/libChronoEngine.dylib  # [osx]
-
+        - test -f ${PREFIX}/lib/libChronoEngine.${SHLIB_EXT}
```

Comment:
I think ${SHLIB_EXT} evaluates to `.so` etc. so the dot needs to go away :)

Reply:
Thanks, I pushed the fix.